solutions/instances/variables.tf

```
@@ -123,6 +123,12 @@ variable "cloud_monitoring_provision" {
   default     = true
 }
 
+variable "existing_cloud_monitoring_crn" {
+  description = "The CRN of an IBM Cloud Monitoring instance, if value is provided, then var 'cloud_monitoring_provision' should be false."
```

```suggestion
  description = "The CRN of an IBM Cloud Monitoring instance. If specified, set the `cloud_monitoring_provision` variable to `false`."
```

And then, update `cloud_monitoring_provision`

```
description = "Whether to create an IBM cloud monitoring instance. Set to `false` if a CRN is specified in `existing_cloud_monitoring_crn`."
```

---

addressed